### PR TITLE
Feature/frontend build pipeline

### PR DESCRIPTION
**Pull request by loic294**
# Description

**Make sure the PR is against the `develop` branch (see [Contributing](https://github.com/nccgroup/ScoutSuite/blob/master/CONTRIBUTING.md)).**

Please include a summary of the change(s) and which issue(s) it addresses. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change

Select the relevant option(s):

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (optional)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/frontend-test.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: Test the Fontend Build
+
+on:    
+  # Trigger the test when a PR is created for develop
+  pull_request: 
+    branches:
+      - v6-master # TO REMOVE - For testing only
```
Comment (j4v) [inline .github/workflows/frontend-test.yml:7]: Should this still be here?

File: .github/workflows/frontend-build.yml [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 
-on:
-  # Triggers the workflow on push or pull request events but only for the master branch
-  push:
-    branches: [ master, v6-master ]
+on:    
+  # Trigger the test when a PR is created for develop
+  push: 
+    branches:
+      - v6-master # TO REMOVE - For testing only
```
Comment (j4v) [inline .github/workflows/frontend-build.yml:7]: Should this still be here?

File: .github/workflows/frontend-test.yml (hunk 1 above)
Comment (loic294) [reply .github/workflows/frontend-test.yml]: Removed!

File: .github/workflows/frontend-build.yml (hunk 2 above)
Comment (loic294) [reply .github/workflows/frontend-build.yml]: Removed!

**Pull request merged**